.travis.yml

```diff
@@ -1,14 +1,21 @@
 ---
 language: java
+# skip the default install (mvn install -DskipTests=true) to speed up the build
+install: true
 script: mvn -q install -P examples
 jdk:
 - openjdk7
 matrix:
   include:
   - jdk: oraclejdk7
-    script: mvn -q deploy --settings .travis-settings.xml -Dno.gem.deploy=true
+    # skip the default install (mvn install -DskipTests=true) to speed up the build
```

Comment:
You are right, it is inherited from above, because that oraclejdk8 job did execute `true` for install (even though just the comment was specified on the job itself).